ghost/core/core/server/data/migrations/versions/5.93/2024-09-03-20-09-40-null-analytics-jobs-timings.js

```diff
@@ -0,0 +1,15 @@
+// For information on writing migrations, see https://www.notion.so/ghost/Database-migrations-eb5b78c435d741d2b34a582d57c24253
```

Comment:
I think we need to change the path to the migration, since we released 5.93 today 😅 I had to change it to `/versions/5.94/asdfasdfasd` to get this to run locally

Reply:
Updated, good catch.